support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -96,4 +96,28 @@ export const tests: Tests = {
     targetPage: usLandingPage,
     seed: 13,
   },
+
+  landingPagePriceBreakdownTest: {
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'daily',
+      },
+      {
+        id: 'none',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    referrerControlled: false,
+    targetPage: allLandingPagesAndThankyouPages,
```

Comment:
I'm going to investigate after lunch if this regex is doing what we want it to do

Reply:
It works but it's more complex than it needs to be. `/??` = 0 or 1 `/`s, none greedily - as we're following it directly with another `/` it's always matching on 0.